docs/theming/introduction.mdx

```diff
@@ -4,36 +4,84 @@ sidebar_position: 1
 title: Introduction
 ---
 
-import ChannelPreviewThemeScreenshot1 from '../assets/stream-chat-css-channel-preview-theme1-screenshot.png';
-import ChannelPreviewThemeScreenshot2 from '../assets/stream-chat-css-channel-preview-theme2-screenshot.png';
-import ChannelListScreenshot from '../assets/stream-chat-css-channel-list-screenshot.png';
-import ChannelListDarkScreenshot from '../assets/stream-chat-css-channel-list-dark-screenshot.png';
-import ChannelListCustomizedScreenshot from '../assets/stream-chat-css-channel-list-customized-screenshot.png';
-import ChannelListLayoutScreenshot from '../assets/stream-chat-css-channel-list-layout-screenshot.png';
-import ChannelListRtlScreenshot from '../assets/stream-chat-css-channel-list-rtl-screenshot.png';
+import SDKSpecific from './SDKSpecific';
+import ChatUiScreenshot from '../assets/stream-chat-css-chat-ui-screenshot.png';
+import DarkUiScreenshot from '../assets/stream-chat-css-dark-ui-screenshot.png';
+import CustomDarkUiScreenshot from '../assets/stream-chat-css-custom-dark-theme-screenshot.png';
+import ChatUiThemeCustomizationScreenshot from '../assets/stream-chat-css-chat-ui-theme-customization-screenshot.png';
+import ChatUiLayoutScreenshot from '../assets/stream-chat-css-chat-ui-layout-screenshot.png';
+import CustomAvatarColorScreenshot from '../assets/stream-chat-css-custom-avatar-color-screenshot.png';
+import MessageScreenshot from '../assets/stream-chat-css-message-color-screenshot.png';
+import MessageCustomColorScreenshot from '../assets/stream-chat-css-message-color-customization-screenshot.png';
+import MessageCustomColor2Screenshot from '../assets/stream-chat-css-message-color-customization2-screenshot.png';
+import ChatUiSquareThemeScreenshot from '../assets/stream-chat-css-square-theme-screenshot.png'
+import ChatUiRtlScreenshot from '../assets/stream-chat-css-rtl-layout-screenshot.png'
 
 :::info
-The default chat UI has a new theming and customization system. This page contains information about the new version (version 2). The most significant improvements of the new version:
+The SDK has a new theming and customization system. This page contains information about the new version (version 2). The most significant improvements of the new version:
 
 - Refreshed design
 - Better customization
 - Support for RTL layout
 
 The [old theme](../concepts/theming-and-css.mdx) can still be used with the latest SDK versions, but it's now deprecated and will be removed in a future release.
+
+<SDKSpecific name="angular">
+
+To use the new theme, please upgrade to stream-chat-angular version 4 and follow the instructions below.
```

Comment:
We have to leave empty lines before and after content to have proper Markdown formatting inside `SDKSPecific` component